api/models.py

```diff
@@ -10,6 +10,7 @@ class Tool(db.Model):
     description = db.Column(db.TEXT(65535), nullable=False)
     url = db.Column(db.String(2047), nullable=False)
     tasks = db.relationship("Task", backref="tool", lazy="dynamic")
```

Comment:
The tasks relationship needs to have the `ondelete="CASCADE"` parameter; otherwise, the db won't let us delete tools as long as there are still associated tasks.

Reply:
Addressed with [6830a58](https://github.com/wikimedia/toolhunt/pull/56/commits/6830a58f9774b33f8f3d27d22c1ff949d7864789)